### DISCUSSION TITLE
epmd: invalid packet size

### DISCUSSION THREAD

**Discussion by luffy037** [rabbitmq/rabbitmq-server#6786] [Other]
Can anyone provide any insights into what to look into?
my rabbit cluster is showing some peculiar behavior. Sporadically the whole cluster will hang, and show the following errors in systemctl:
![image](https://user-images.githubusercontent.com/50431806/210308752-42bba04c-336f-4c8b-af0a-886159465ae0.png)
 I searched online for similar errors but couldnt find a single reference or anything even remotely similar.

**Comment by mkuratczyk** [comment]
`erts-10.3.5.4` is from Erlang 21, a version released in the previous decade and not supported by the Erlang team nor compatible with any supported RabbitMQ version. Please upgrade and raise an issue if it still happens.

**Comment by michaelklishin** [comment]
Something (possibly unintentionally) has sent a payload to `epmd` that wasn't what `epmd` expected. `epmd` is [briefly mentioned](https://rabbitmq.com/networking.html#epmd) in the RabbitMQ networking guide but it comes from the runtime (Erlang/OTP).
